client/app.js

```diff
@@ -42,7 +42,6 @@ const porybox = ng.module('porybox', [
 ]);
 
 porybox.controller('MainCtrl', function () {
-  this.boxes = [];
```

Comment:
That statement doesn't do anything since the value immediately gets overwritten by the `Object.assign` statement that follows it.